src/configs.py

```diff
@@ -25,13 +26,21 @@ class CharmConfig:
     def __init__(self, config: ConfigData) -> None:
         self._config = config
 
+    def __getitem__(self, key: str) -> Any:
+        return self._config.get(key)
+
     def to_service_configs(self) -> ServiceConfigs:
         return {
             "dev_mode": self._config["dev"],
             "log_level": self._config["log_level"],
             "access_token_strategy": "jwt" if self._config["jwt_access_tokens"] else "opaque",
         }
 
+    def to_env_vars(self) -> EnvVars:
+        return {
+            "DEV": "true" if self._config["dev"] else "false",
+        }
+
```

Comment:
question: Not sure I get this, do we need to set the env var, the `dev` key in the config and the `--dev` flag in the CLI?